test/test_prototype_transforms_functional.py

```diff
@@ -125,7 +153,7 @@ def test_no_inplace(self, info, args_kwargs, device):
         input_version = input._version
         output = info.kernel(input, *other_args, **kwargs)
 
-        assert output is not input or output._version == input_version
+        assert output is not input or input._version == input_version
```

Comment:
We actually want to check that the `_version` of the `input` has not changed after executing the kernel.